### DISCUSSION TITLE
Are we looking at an EVM (eth based) sidechain for Doge?

### DISCUSSION THREAD

**Discussion by GiverofMemory** [dogecoin/dogecoin#2775] [General]
Bitcoin cash has something called "SmartBCH" which is an Ethereum Virtual Machine" that works as a PoS sidechain to bitcoin cash.  This allows for using metamask and "hop.cash" to convert your BCH to SmartBCH and then wrapped BCH if desired and swap it, (on mistswap or benswap etc) stake it, or use it as liquidity in the ecosystem.

I really like how smartbch works and think it would be a good (open source) model for how dogecoin's mechanism can be.  I would definitly rather launch my token on the dogecoin ecosystem than bitcoin cash because of it's pseudoinflationary emmission will allow the dogecoin marketcap to go up and keep similar EVM fees unlike bitcoin cash where the EVM fees will go up with the price of the coin.

**Comment by rnicoll** [comment]
SmartBCH's on my radar, but requires a new op-code, so would be on the 1.21 base, which is needing me to find more QA from somewhere. So... aware of it, but no roadmap for it right now.

**Reply by GiverofMemory** [reply]
> SmartBCH's on my radar, but requires a new op-code, so would be on the 1.21 base, which is needing me to find more QA from somewhere. So... aware of it, but no roadmap for it right now.

Very cool, ya no rush just wanted to bring that up since I heard about staking ability coming to doge and saw this sort of method as a good way to achieve it.  Also with the evm sidechain would bring token and nft functionality which would open up an entire ecosystem similar to https://www.marketcap.cash which would be pretty neat.
